3_javascript/javascript3/week1/WeatherApp/weather_api.js

```diff
@@ -0,0 +1,12 @@
+const weather_api = "https://api.openweathermap.org/data/2.5/weather?q=copenhagen&appid=d343416b21414202682daff848a54ebe";
+
+async function getWeather() {
```

Comment:
It seems like we are not using this function and this file for anything, so you could just delete it.